sycl/include/sycl/ext/intel/experimental/fpga_kernel_attributes.hpp

```diff
@@ -0,0 +1,154 @@
+//==----- fpga_kernel_attributes.hpp - SYCL properties associated with FPGA
```

Comment:
According to spec,

> an implementation supporting this extension must predefine the macro SYCL_EXT_INTEL_FPGA_KERNEL_INTERFACE_PROPERTIES

Please add this macro to https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/feature_test.hpp.in